index.js

```diff
@@ -920,6 +920,20 @@ const fileType = input => {
 		};
 	}
 
+	if (check([0x4C, 0x00, 0x00, 0x00, 0x01, 0x14, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0xC0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x46])) {
+		return {
+			ext: 'lnk',
+			mime: 'application/x.ms.shortcut'
```

Comment:
Can you add a comment that the MIME is just invented by us? Same with the other one.

Reply:
Well, technically, `x.` already indicates it was just invented by random person. I'll add it to readme.

Reply:
Yes, but sometimes it's a common convention shared by multiple tools/software. Here, it's just *us* using it.